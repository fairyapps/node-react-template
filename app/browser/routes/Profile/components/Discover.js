import React, { Component } from "react";

const Discover = props => <h1>Avatar Here</h1>;

export default Discover;
