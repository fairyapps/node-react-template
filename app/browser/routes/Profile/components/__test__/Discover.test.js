import React from "react";
import renderer from "react-test-renderer";
import Discover from "../About";

test("Discover renders correct snapshot", () => {
  const component = renderer.create(<Discover />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
