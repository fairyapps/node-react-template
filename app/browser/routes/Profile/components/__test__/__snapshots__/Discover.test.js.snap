// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Discover renders correct snapshot 1`] = `
<h1>
  About Here
</h1>
`;
